guardian/interactive-atom-thrasher-template
# Combination thrashers for Australia

## What does this change?

Create a new set of combination thrasher (set 10) for Australian newsletters, including Morning Mail, Afternoon Update and Saved for Later

## How to test

Run gulp from your IDE
Select set-10 from the list of Atoms then select one of the Thrasher containers or Snaps

To test the success/error message has inverted colours for the Saved for Later thrasher, see the testing instructions in PR #39 

## Images

![Screenshot 2022-10-20 at 09 02 55](https://user-images.githubusercontent.com/74301289/196891714-1a7f6bc8-dedd-4a86-b14a-da6d94c24d14.png)
